.github/workflows/ci-latest-helm-chart-release.yaml

```
@@ -0,0 +1,39 @@
+name: ci-latest-helm-chart-release
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "deployments/helm/**"
+
+jobs:
+  publish-chart:
+    name: Update Stable Helm Chart With Latest Changes
+    if: ${{ (github.repository == 'delusionaloptimist/kubearmor') && (!contains(github.event.head_commit.message, '[Automated]')) }}
```

```suggestion
    if: ${{ (github.repository == 'kubearmor/kubearmor') && (!contains(github.event.head_commit.message, '[Automated]')) }}
```

Let's handle this before merging, Do we need to do any steps for helm repo setup for KubeArmor which I can help with?

---

Good eye.

---

@daemon1024 Can you create the `GH_ACCESS_TOKEN` secret? It'll be a PAT with read and write permissions to kubearmor repos

---

@DelusionalOptimist why do we need a PAT, why is GH Action token not enough?

---

@daemon1024 AFAIK, the default GITHUB_TOKEN doesn't give enough permissions to push changes across repositories.

---

@daemon1024 Here is a failed sample run using GH_TOKEN - https://github.com/DelusionalOptimist/KubeArmor/actions/runs/5335563256/jobs/9668964507#step:6:39